connectivity/check/test.go

```diff
@@ -129,6 +153,8 @@ type Test struct {
 	warnBuf *bytes.Buffer
 	verbose bool
 
+	conditionFn func() bool
```

Comment:
```suggestion
        // conditionFn is a function that returns true if the test needs to run,
        // and false otherwise. By default, it's set to a function that returns
        // true.
	conditionFn func() bool
```

Reply:
Fixed.